mobile/lib/pages/editing/crop.page.dart

```diff
@@ -25,13 +25,14 @@ class CropImagePage extends HookWidget {
 
     return Scaffold(
       appBar: AppBar(
-        backgroundColor: Theme.of(context).bottomAppBarTheme.color,
-        leading: CloseButton(color: Theme.of(context).iconTheme.color),
+        backgroundColor: Theme.of(context).scaffoldBackgroundColor,
+        title: const Text('Crop'),
+        leading: CloseButton(color: Theme.of(context).primaryColor),
         actions: [
           IconButton(
             icon: Icon(
               Icons.done_rounded,
-              color: Theme.of(context).iconTheme.color,
+              color: Theme.of(context).primaryColor,
```

Comment:
You can do `context.primaryColor` I believe

Reply:
Done!